File: web3rpc/rpc-specs/code-samples/python/klay/miscellaneous/sha3.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

hashData = "0x11223344"

w3 = Web3(Web3.HTTPProvider(host))
klay_response = w3.klay.sha3(hashData)

print(klay_response)

File: web3rpc/rpc-specs/code-samples/javascript/klay/miscellaneous/sha3.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");

  const data = "0x11223344";

  provider.klay
    .sha3(data, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();
